package.json

```diff
@@ -57,7 +57,7 @@
     "react-animate-height": "^2.0.20",
     "react-animations": "^1.0.0",
     "react-circular-progressbar": "^2.0.3",
-    "react-device-frame": "^2.1.4",
```

Comment:
no longer need this library which was used for phone preview. we made our own phone preview based on Sasha design